### ISSUE TITLE
send request to lissi backend and create a digital identity

### ISSUE THREAD

**Issue by veitmo** [amosproj/amos2022ss04-digital-identity#25]
## User story
1. As a {HR-employee}
2. I want / need {the DIdentity backend to connect to the Lissi backend in order to create a digital identity}
3. So that {the process of creating a digital identity is carried out}

## Acceptance criteria
* DIdentity backend has connection to Lissi backend
* all necessary information (name, surname, email-adress) for creation of digital identity is sent to Lissi backend
* receive corresponding URL to the newly created DI

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release

**Comment by veitmo** [comment]
@jackDS008 
Hi Jannik, we have discussed in the last sprint meeting that the public keys (some identification number) for each created digital identity must be stored in our backend/database. We weren't aware of that during the creation of this issue. I assume that the storing-process of the key is not included in this issue, correct? Then I will create a new issue for this step. 
If the storing-process is already included, even better. Then, I will update the acceptance criteria

**Comment by Idontker** [comment]
Done in sprint 5 https://github.com/amosproj/amos2022ss04-digital-identity/commit/8b9cdf974b0592bb76a4d87a3d35fe9834c38272
